content/guides/dashboard/gitlab-integration.md

```diff
@@ -20,25 +20,41 @@ If you are still facing issues after this, please
 
 </Alert>
 
+<Alert type="warning">
+
+GitLab Self-managed Integration is dependent on your GitLab instance being able
+to reach our Dashboard API on the open internet. If your instance is running
+behind a firewall, you may need to reconfigure your security infrastructure to
+allow communication with our servers.
+
```

Comment:
@cy-veronica  will provide users with a list of IPs to whitelist in their infrastructure.

Reply:
[Internal comment about this question.](https://cypressio.slack.com/archives/C02HH608ZT5/p1643672421455569)